wasm2c/wasm-rt.h

```diff
@@ -39,24 +39,32 @@ extern "C" {
  *
  * This is usually 10%-25% faster, but requires OS-specific support.
  * */
+// #define WASM_RT_MEMCHECK_SIGNAL_HANDLER 1
```

Comment:
remove comment?

Reply:
I was using it as documentation of what the flag is. Maybe I could make that more clear?

Reply:
Maybe it inside the docstring? without the 1?

Something like `Set the default value for WASM_RT_MEMCHECK_SIGNAL_HANDLER which determines ...`?

Reply:
Done.